cs130_front/src/components/views/GroupsPage/GroupsPage.js

```diff
@@ -1,35 +1,105 @@
 import React, { useState, useEffect } from 'react';
+import ClassList from '../../CurrentClasses/ClassList';
+import GroupItem from './GroupItem';
+import Text from '../../Text/Text';
+import Button from '../../Button/Button';
+import Requests from '../../Requests/Requests';
+import * as Colors from '../../../constants/Colors';
 import UserList from '../../UserList/UserList';
 import '../../../App.css';
+import CreateGroup from './CreateGroup';
 
 function GroupsPage(props) {
 
 
-  const classes = [
-    {name: "Discrete Mathematics", courseId: 1, groups: [{name: "DM Squad", groupId: 1}, {name: "DM Squad II", groupId: 1}] }, 
-    {name: "Computer Architecture", courseId: 2, groups: [{name: "CA Squad", groupId: 3}]}
+  const classes2 = [
+    { name: "Discrete Mathematics", courseId: 1, groups: [{ name: "DM Squad", groupId: 1 }, { name: "DM Squad II", groupId: 1 }] },
+    { name: "Computer Architecture", courseId: 2, groups: [{ name: "CA Squad", groupId: 3 }] }
   ]
-  const [mainPanelState, setMainPanel] = useState();
 
-  const members= [
-    {name: "Shirly fang", school: "UCLA", id:123, discord:"shirly#123", email:"shirly@gmail.com"},
-    {name: "Shirly fang", id:123, discord:"shirly#123", email:"shirly@gmail.com"}
+  const members = [
+    { name: "Shirly fang", school: "UCLA", id: 123, discord: "shirly#123", email: "shirly@gmail.com" },
+    { name: "Shirly fang", id: 123, discord: "shirly#123", email: "shirly@gmail.com" }
   ]
+
+  const currentGroup2 = { id: 123, name: "DM Squad", courseName: "Calculus", members: members, day: "friday", time: "4:30pm" }
```

Comment:
Let's redo all of this to look like this 
```
  const members = [
    { name: "Shirly fang", school: "UCLA", id: 123, discord: "shirly#123", email: "shirly@gmail.com" },
    { name: "Shirly fang", id: 123, discord: "shirly#123", email: "shirly@gmail.com" }
  ]

  const currentGroup2 = { id: 123, name: "DM Squad", courseName: "Calculus", members: members, day: "friday", time: "4:30pm" }

  const classes2 = [
    { name: "Discrete Mathematics", courseId: 1, groups: [ currentGroup2 ], },
    { name: "Computer Architecture", courseId: 2, groups: [currentGroup2] }
  ]
```

This is because when we network, in order to get the classeslist and populate the rest of the group page properly, we need classes to be in this format.